docker/confd/templates/parameters.yml.tmpl

```diff
@@ -22,8 +21,15 @@ parameters:
             lastname: {{ getv (printf "/fixtures/%s/lastname" .) }}
             password: {{ getv (printf "/fixtures/%s/password" .) }}
             roleId: {{ getv (printf "/fixtures/%s/roleid" .) }}
+            roleName: {{ getv (printf "/fixtures/%s/rolename" .) }}
```

Comment:
we might not need this, but until it's not all merged, I think we need to wrap this with an "if" as well, not sure about the merge order atm